android-components/components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarPresenter.kt

```diff
@@ -59,7 +59,13 @@ class ToolbarPresenter(
         val tab = state.findCustomTabOrSelectedTab(customTabId)
 
         if (tab != null) {
-            renderer.post(tab.content.url)
+            // If the search terms should be displayed and the tab has search terms set,
+            // then display the search terms
```

Comment:
```suggestion
            // If the search terms should be displayed and the tab has search terms set,
            // then display the search terms.
```
This comment might just be repeating what the code is already doing. I would lean towards just not having it.